main.py

```diff
@@ -1 +1,23 @@
 #Mad Libs Generator Project
+//Loop back to this point once code finishes
```

Comment:
We suggest converting "//" to "#" for these comments. "//" is not Pythonic. Same comment for lines 5, 12 and 22.